package.json

```diff
@@ -21,21 +12,21 @@
     "packages/prettier-config-local"
   ],
   "scripts": {
-    "build": "yarn workspaces run build",
-    "lint": "yarn workspaces run lint",
-    "pretty": "yarn workspaces run pretty",
+    "lint": "yarn workspaces run lint && prettier --ignore-path .prettierignore-ci --check .",
+    "pretty": "yarn workspaces run pretty && prettier --ignore-path .prettierignore-ci --write .",
```

Comment:
This is unnecessary, we could simply run `"prettier --write ."` and remove `.prettierignore-ci`.